notebooks/introduction_to_optimum_graphcore.ipynb

```diff
@@ -63,7 +63,7 @@
    "outputs": [],
    "source": [
     "%pip install optimum[graphcore]\n",
-    "%pip install transformers==4.20.0"
+    "%pip install optimum-graphcore==0.4.1"
```

Comment:
I think we want to be a bit looser, this will also create some maintenance as we update optimum

```suggestion
    "%pip install \"optimum-graphcore>0.4, <0.5\""
```

Implicitely in paperspace we'll pre-install the correct version so that shouldn't be as much of a problem the requirement will be already satisfied.